tests/manage/test_dynamic_pvc_accessmodes_with_reclaim_policies.py

```diff
@@ -295,10 +282,158 @@ def finalizer():
+            desired_status=constants.STATUS_RUNNING, wait=True,
+            namespace=self.namespace, node_name=self.worker_nodes_list[1],
+            pod_dict_path=constants.NGINX_POD_YAML
+        )
+        node_pod2 = pod_obj2.get().get('spec').get('nodeName')
+
+        assert node_pod1 != node_pod2, 'Both pods are on the same node'
+
+        # Run IO on both the pods
+        logger.info(f"Running IO on pod {pod_obj1.name}")
+        file_name1 = pod_obj1.name
+        logger.info(file_name1)
+        pod_obj1.run_io(
+            storage_type=self.storage_type, size=self.io_size, runtime=30,
+            fio_filename=file_name1
+        )
+
+        logger.info(f"Running IO on pod {pod_obj2.name}")
+        file_name2 = pod_obj2.name
+        pod_obj2.run_io(
+            storage_type=self.storage_type, size=self.io_size, runtime=30,
+            fio_filename=file_name2
+        )
+
+        # Check IO and calculate md5sum of files
+        pod.get_fio_rw_iops(pod_obj1)
+        md5sum_pod1_data = pod.cal_md5sum(
+            pod_obj=pod_obj1, file_name=file_name1
+        )
+
+        pod.get_fio_rw_iops(pod_obj2)
+        md5sum_pod2_data = pod.cal_md5sum(
+            pod_obj=pod_obj2, file_name=file_name2
+        )
+
+        logger.info(f"verify data from alternate pods")
+
+        assert pod.verify_data_integrity(
+            pod_obj=pod_obj2, file_name=file_name1,
+            original_md5sum=md5sum_pod1_data
+        )
+
+        assert pod.verify_data_integrity(
+            pod_obj=pod_obj1, file_name=file_name2,
+            original_md5sum=md5sum_pod2_data
+        )
+
+        # Verify that data is mutable from any pod
+
+        logger.info(f"Perform modification of files from alternate pod")
+        # Access and rename file written by pod-2 from pod-1
+        file_path2 = pod.get_file_path(pod_obj2, file_name2)
+        logger.info(file_path2)
+        pod_obj1.exec_cmd_on_pod(
+            command=f"bash -c \"mv {file_path2} {file_path2}-renamed\"",
+            out_yaml_format=False
+        )
+
+        # Access and rename file written by pod-1 from pod-2
+        file_path1 = pod.get_file_path(pod_obj1, file_name1)
+        logger.info(file_path1)
+        pod_obj2.exec_cmd_on_pod(
+            command=f"bash -c \"mv {file_path1} {file_path1}-renamed\"",
+            out_yaml_format=False
+        )
+
+        logger.info(f"Verify presence of renamed files from both pods")
+        file_names = [f"{file_path1}-renamed", f"{file_path2}-renamed"]
+        for file in file_names:
+            assert pod.check_file_existence(pod_obj1, file), (
+                f"File {file} doesn't exist"
+            )
+            logger.info(f"File {file} exists in {pod_obj1.name} ")
+            assert pod.check_file_existence(pod_obj2, file), (
+                f"File {file} doesn't exist"
+            )
+            logger.info(f"File {file} exists in {pod_obj2.name}")
+
+        pod_obj1.delete()
```

Comment:
move pods deletion to the teardown as explained in the above comment

Reply:
Done.